### PR TITLE
Keycloak 6640

### PR DESCRIPTION
**Pull request by matthewhelmke**
Per https://issues.jboss.org/browse/KEYCLOAK-6640 and https://issues.jboss.org/browse/RHSSO-1273.

### CODE REVIEW

File: server_installation/topics/network/https.adoc [hunk 1]
```diff
@@ -119,10 +119,26 @@ $ keytool -import -alias yourdomain -keystore keycloak.jks -file your-certificat
 ===== Configure {project_name} to Use the Keystore
 
 Now that you have a Java keystore with the appropriate certificates, you need to configure your {project_name} installation to use it.
-First step is to move the keystore file to the _configuration/_ directory of your deployment and to edit the _standalone.xml_, _standalone-ha.xml_ or _domain.xml_ file to use
-the keystore and enable HTTPS.  (See <<_operating-mode, operating mode>>).
+First, you must move the keystore file to the _configuration/_ directory of your deployment and edit the _standalone.xml_, _standalone-ha.xml_, or _host.xml_ file to use the keystore and enable HTTPS.  (See <<_operating-mode, operating mode>>).
```
Comment (slaskawi) [inline server_installation/topics/network/https.adoc]: This is not strictly required to be honest. You may use an absolute path and point to a specific file on your system. Using a relative path is just for simplification. 

Perhaps we should rephrase it a little bit to take what I mentioned into consideration?

**Review by slaskawi — DISMISSED** [review]
LGTM, just a small nitpick.

File: server_installation/topics/network/https.adoc (hunk 1 above)
Comment (stianst) [reply server_installation/topics/network/https.adoc]: @matthewhelmke can you respond please?

Comment (matthewhelmke) [reply server_installation/topics/network/https.adoc]: Apologies, I missed this comment. What do you think of this, @slaskawi and @stianst? 

First, you must edit the _standalone.xml_, _standalone-ha.xml_, or _host.xml_ file to use the keystore and enable HTTPS.  You may then either move the keystore file to the _configuration/_ directory of your deployment or the file in a location you choose and provide an absolute path to it. (See <<_operating-mode, operating mode>>).

My question is where the user provides that absolute path info; I am either missing it or not seeing where that is configured anywhere in this file. If we don't document that here in this same document, I would prefer to be opinionated and simply tell them to use the default location. Too many options makes installation much more difficult.

Comment (slaskawi) [reply server_installation/topics/network/https.adoc]: That sounds good to me. 

The `relative-to` parameter is optional. If you remove it, you can (or even should) use absolute paths.

Comment (matthewhelmke) [reply server_installation/topics/network/https.adoc]: Okay. I'll update shortly with a new commit.

Comment (matthewhelmke) [reply server_installation/topics/network/https.adoc]: Sorry, I seem to have messed up my local branch and cannot push a new commit. This is what I am trying to add:

First, you must edit the _standalone.xml_, _standalone-ha.xml_, or _host.xml_ file to use the keystore and enable HTTPS. You may then either move the keystore file to the _configuration/_ directory of your deployment or the file in a location you choose and provide an absolute path to it. If you are using absolute paths, remove the optional `relative-to` parameter from your configuration (See <<_operating-mode, operating mode>>).

**Comment by matthewhelmke** [comment]
Hmm. I had to do a `git pull` in order to push because my local branch was behind this one. Looks okay to me now.

File: server_installation/topics/network/https.adoc (hunk 1 above)
Comment (slaskawi) [reply server_installation/topics/network/https.adoc]: You could try to fix this by using `git reset --soft upstream/master` (that will reset your git tree to `upstream/master` but leave the modified files), then `git commit <...> && git push <...>`.

LGTM

**Review by slaskawi — APPROVED** [review]

**Review by abstractj — APPROVED** [review]

**Pull request merged**